File: 1913-maximum-product-difference-between-two-pairs/1913-maximum-product-difference-between-two-pairs.java
class Solution {
    public int maxProductDifference(int[] nums) {
        Arrays.sort(nums);
        int size = nums.length;
        int high = nums[nums.length-1] * nums[nums.length-2];
        int low = nums[0] * nums[1];
        int sol = high - low;
        return sol;
    }
}

File: 1913-maximum-product-difference-between-two-pairs/1913-maximum-product-difference-between-two-pairs.cpp
#include <algorithm>
class Solution {
public:
    int maxProductDifference(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int high = nums[nums.size()-1] * nums[nums.size()-2];
        int low = nums[0]*nums[1];
        int ans = high - low;
        return ans;
    }
};